model/src/main/proto/arguments.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// The fragment arguments passed to instances of ExitProfileDialogFragment.
+message ExitProfileDialogArguments {
+  // Decides the correct menu item to be highlighted after canceling the ExitProfile dialog.
+  oneof highlight_item {
+    // Whether any menu item needs to be highlighted or not.
+    bool highlight_none = 1;
+
+    // Whether some menuItem from the Navigation Drawer menu needs to be highlighted or not.
+    bool highlight_last_checked_menu_item = 2;
+
+    // Whether Administrator Controls item from the Navigation Drawer footer needs to be
+    // highlighted or not.
+    bool highlight_administrator_controls_item = 3;
```

Comment:
I also had a second thought to create an enum in place of these two, it may look more cleaner because they both are mutually exclusive (but not exhaustive).

Reply:
I was going to ask the same thing, I think an enum seems to work better here

Reply:
Please confirm if you are referring to the below model - 
```
oneof HighlightItem {
     bool highlight_none = 1;
     HighlightCorrespondingItem highlight_corresponding_menu_item = 2;
}

enum HighlightCorrespondingItem {
     HIGHLIGHT_LAST_CHECKED_MENU_ITEM_UNSPECIFIED  = 0;
     HIGHLIGHT_ADMINISTRATOR_CONTROLS_ITEM = 1;
}
```
If yes then can you please suggest some good variable names here? Like what should be a good alternative for `HighlightCorrespondingMenuItem`?

Reply:
No I actually meant something like:
```
enum HighlightItem {
NONE = 0;
LAST_CHECKED_MENU = 1;
ADMINISTRATOR_CONTROLS = 2;
}
```

Something on those lines

Reply:
Done.